run-smoke-tests.sh

```diff
@@ -0,0 +1,57 @@
+#!/bin/bash
+
+# https://repo.maven.apache.org/maven2/io/thorntail/bom/
+THORNTAIL_VERSION=2.7.0.Final
+OUTPUT_DIRECTORY=./failsafe-reports
+POSITIONAL=()
+while [[ $# -gt 0 ]]
+do
+key="$1"
+
+case $key in
+    -v|--version.io.thorntail)
+    THORNTAIL_VERSION="$2"
+    shift
+    shift
+    ;;
+    -o|--output)
+    OUTPUT_DIRECTORY="$2"
+    shift
+    shift
+    ;;
+    -h|--help)
+    echo "This is script for running Thorntail OCP smoke tests"
+    echo "-v|--version.io.thorntail This parameter specify the Thorntail version. The default is ${THORNTAIL_VERSION}"
+    echo "-o|--output This parameter specify output directory. The default is ${OUTPUT_DIRECTORY}"
+    echo "-h|--help This parameter print this message"
+    exit 1;
+    shift
+    shift
```

Comment:
You probably don't need the `shift`s here, since you just exited :-) (And it looks wrong. `-h` doesn't have a value. If anything, there should be just one `shift`.)